solr/core/src/java/org/apache/solr/cli/CLIUtils.java

```diff
@@ -322,7 +322,7 @@ public static boolean safeCheckCoreExists(String solrUrl, String coreName, Strin
           Thread.sleep(clamPeriodForStatusPollMs);
         }
         final var coreStatusReq = new CoresApi.GetCoreStatus(coreName);
-        final var coreStatusRsp = coreStatusReq.process(solrClient).getParsed();
```

Comment:
I admit the result of this work as seen in consuming/calling code right here is _merely_ removing `getParsed()` since there is now no intermediate class holding the parsed value.  But anyone browsing the generated code / class hierarchy will now see something simpler.